src/ucs/sys/module.c

```diff
@@ -49,7 +57,7 @@ static void ucs_module_loader_add_dl_dir()
         return;
     }
 
-    ucs_debug("ucs library path: %s", dl_info.dli_fname);
+    ucs_module_debug("ucs library path: %s", dl_info.dli_fname);
```

Comment:
maybe it is worth to be logged as info, so it would be available in the release build?